pages/common/speedtest-cli.md

```diff
@@ -1,32 +1,32 @@
 # speedtest-cli
 
-> Command line interface for testing internet bandwidth using speedtest.net.
+> Command line interface for testing internet bandwidth using https://speedtest.net.
 > More information: <https://github.com/sivel/speedtest-cli>.
 
 - Run a speed test:
 
 `speedtest-cli`
 
-- Run without performing download tests:
+- Run a speed test without performing download tests:
 
 `speedtest-cli --no-download`
 
-- Run a speed test and generate a shareable result picture:
+- Run a speed test and generate an image of the results (useful for sharing):
 
 `speedtest-cli --share`
 
-- Print a list of all speedtest.net servers, sorted by distance, to file:
-
-`speedtest-cli --list > speedtest_servers.txt`
-
 - Run a speed test to the given speedtest.net server id:
 
 `speedtest-cli --server {{server_id}}`
 
-- Display results in CSV format, suppressing the progress information:
+- Save a list of all speedtest.net servers, sorted by distance:
+
+`speedtest-cli --list > {{speedtest_servers.txt}}`
+
+- Display results as CSV (suppressing progress information):
 
 `speedtest-cli --csv --csv-delimiter {{delimiter_char}}`
 
-- Display results in JSON format, suppressing the progress information:
+- Display results as JSON (suppressing progress information):
```

Comment:
I think `JSON format` is a favorite from @sbrl lol

Reply:
This change is fine as-is I think. The `as JSON` is quite an elegant way of putting it :slightly_smiling_face: